src/index.html

```diff
@@ -2,11 +2,384 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+        class="logo"
+        src="./images/logo.png"
+        alt="Moyo logo"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <section class="section">
```

Comment:
```suggestion
      <section class="content">
```
'section' is not very meaningful name